roommanager.js

```diff
@@ -19,17 +19,22 @@ module.exports = function (server, storage) {
 			users: [],
 		};
 
-
 		for (let i = 0; i < room.clients.length; i++) {
```

Comment:
I know I'm sounding like a broken record here, but this really should be a forEach loop, especially in this section of code. The forEach function allows the loop to run asynchronously, while normal for loops block. As we are dealing with websockets here and not just arrays, this is a performance issue and not simply a styling one.

My comment also applies to the loop syncing clients down below

Reply:
The purpose of this PR in particular is to fix a bug. I'd be happy to refactor in a separate PR, but I want this fixed because production goes down quite frequently due to this bug.